runtime/test/schema-tests.js

```diff
@@ -211,13 +211,28 @@ describe('schema', function() {
     let References = manifest.findSchemaByName('References').entityClass();
     
     let ReferencedOneSchema = manifest.findSchemaByName('ReferencedOne');
-    assert.doesNotThrow(() => {new References({one: new Reference({id: 'test', storageKey: 'test'}, Type.newEntity(ReferencedOneSchema), null), two: null}); });
-    assert.throws(() => {new References({one: null, two: new Reference({id: 'test', storageKey: 'test'}, Type.newEntity(ReferencedOneSchema), null)}); }, TypeError,
+    assert.doesNotThrow(() => {new References({one: new Reference({id: 'test', storageKey: 'test'}, Type.newReference(Type.newEntity(ReferencedOneSchema)), null), two: null}); });
+    assert.throws(() => {new References({one: null, two: new Reference({id: 'test', storageKey: 'test'}, Type.newReference(Type.newEntity(ReferencedOneSchema)), null)}); }, TypeError,
                   `Cannot set reference two with value '[object Object]' of mismatched type`);
     assert.throws(() => {new References({one: 42, two: null}); }, TypeError,
                   `Cannot set reference one with non-reference '42'`);
   });
 
+  it('collection types', async function() {
```

Comment:
describe the test better? I believe the test name should be a sentence e.g. it('can do some magical thing', ...)

maybe it('enforces the type of the collections of references' ?

Reply:
Done.